sdk/test/Performance/CborPerformanceBenchmarks/CborPerformanceBenchmarksRunner/CloudWatchBenchmarks/ListMetricsBenchmarks.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Amazon;
+using Amazon.CloudWatch;
+using Amazon.CloudWatch.Model;
+using Amazon.CloudWatch.Model.Internal.MarshallTransformations;
+using Amazon.Runtime;
+using BenchmarkDotNet.Attributes;
+
+namespace CborPerformanceBenchmarksRunner.CloudWatchBenchmarks;
+
+public class ListMetricsBenchmarks : BaseBenchmarks
+{
+    [Params(0)]
+    public override int DimensionValue { get; set; }
+
+    public override string Protocol { get; } =
+#if USE_CBOR
+        "CBOR";
+#else
+        "Query";
+#endif
+
+    public string SuiteId { get; set; } = Guid.NewGuid().ToString();
+    public DateTime BaseTime { get; set; } = DateTime.Now.AddHours(-2);
+
+    public override string Service { get; } = "CloudWatch";
+    public override string TestCase { get; } = "List metrics";
+
+    protected IAmazonCloudWatch _client;
+    public override void CreateServiceClient()
+    {
+        _client = new AmazonCloudWatchClient(new AmazonCloudWatchConfig
+        {
+            DisableRequestCompression = true,
+            MaxErrorRetry = 0,
+            RegionEndpoint = RegionEndpoint.USWest2,
+        });
+    }
+
+    private ListMetricsRequest request;
+
+    [Benchmark(Description = "Serialization time (ms)")]
+    public override byte[] Marshall()
+    {
+        var iRequest = ListMetricsRequestMarshaller.Instance.Marshall(request);
+        MarshalledRequest = iRequest;
+
+        return iRequest.Content;
+    }
+
+    [Benchmark(Description = "Total request time (ms)")]
+    public async override Task<AmazonWebServiceResponse> TotalRequest()
+    {
+        var response = await _client.ListMetricsAsync(request);
+        ResponseSizeBytes = Math.Max(ResponseSizeBytes, response.ContentLength);
+
+        return response;
+    }
+
+    public override void GenerateRequest()
+    {
+        request = new ListMetricsRequest
+        {
+            Namespace = "TestNamespace",
```

Comment:
Isn't the result going to be empty with a custom namespace?

I know it depends on the account, but isn't there a value we can use that'll return some values (and actually exercise the marshallers)? From some testing `aws cloudwatch list-metrics --namespace AWS/Usage` returns results even in an account I don't use often.